app/views/items/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<center>
+  <section>
+    <div id="item-show">
+      <h1><a href="/items/<%=@item.id%>"><%= @item.name %></a></h1>
+    </div>
+    <div class="">
+      <img src=<%=@item.image %>>
+    </div>
+    <div class="">
+      <p><%= @item.description %></p>
+      <p>Price: <%= number_to_currency(@item.price) %></p>
+      <p>Number Remaining: <%= @item.inventory %></p>
+      <p>Active: <%= @item.active? %></p>
+      <p>Where to buy: <a href="/merchants/<%=@item.merchant_id%>"><%= Merchant.find(@item.merchant_id).name %>
```

Comment:
You have some logic in here that you don't need. To just make a link for the merchant show page by using their name, you can do 
```html.erb 
<%= link_to @item.merchant.name, "/merchants/#{@item.merchant.id}" %>
```

My guess is that you probably wrote this view before realizing how to write links.